input/fsh/Examples.fsh

```diff
@@ -347,3 +347,29 @@ Description: "mCODE Example for TNM Pathological Regional Nodes Category"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * valueCodeableConcept = AJCC#pN3 "N3"
+
+Instance: mCODECancerRelatedMedicationRequestExample01
+InstanceOf: CancerRelatedMedicationRequest
+Description: "mCODE Example for CancerRelatedMedicationRequest"
+* id = "mCODECancerRelatedMedicationRequestExample01"
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-cancer-related-medication-request"
+* subject = Reference(mCODEPatientExample01)
+* status = MedRequestStatus#active
+* intent = MedRequestIntent#order
+* authoredOn = "2019-04-01"
+* medicationCodeableConcept = RXN#349472 "gefitinib 250 MG Oral Tablet"
+* reasonCode = SCT#254637007 "Non-small cell lung cancer (disorder)"
```

Comment:
Normally one wouldn't have both reasonCode and reasonReference.

Reply:
I don't think that's true but if you want it that way, I'll change it.